package.json

```diff
@@ -56,7 +56,7 @@
     "lint": "tslint --project tslint.json",
     "build": "node_modules/typescript/bin/tsc --rootDir ./ --outDir .build --sourceMap false && npm run build:copy",
     "build:copy": "find src -type f \\( -name \"*.yml\" -or -name \"*.json\" \\) | cpio -pdm .build && find specs -type f \\( -name \"*.yml\" -or -name \"*.json\" -or -name \"*.base64\" \\)  | cpio -pdm .build",
-    "security-scan": "git secrets --scan && git log -p | scanrepo",
+    "security-checks": "git secrets --scan && git log -p | scanrepo",
```

Comment:
I am a bit confused so apologies for the silly question,
It seems that other repos run security-scan that will call git secrets --scan && git log -p | scanrepo however this one has the script named security-scan.
Have we got a different name for this script in the pipeline ?
I was reviewing this one too: https://github.com/dvsa/cvs-svc-app-logs/pull/16/files